cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -691,3 +691,80 @@ subjects:
   namespace: rook-ceph
 # OLM: END CMD REPORTER ROLEBINDING
 ---
+#
+# OLM: BEGIN CLUSTER ROLE
+---
+# Aspects for creation of monitoring resources
+kind: ClusterRole
```

Comment:
I don't think we should be giving Rook blanket permissions to change monitoring in all namespaces. This should just be a Role which allows Rook to change monitoring in only the needed namespace.